apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -1275,6 +1955,93 @@ module.exports = {
       name: "settings",
       label: "Settings",
       files: [
+        {
+          name: "general",
+          label: "General",
+          file: `${APP_DIRECTORY}content/settings/general.md`,
+          fields: [
+            {
+              label: "SEO",
+              name: "seo",
+              widget: "object",
+              fields: [
+                {
+                  label: "Title",
+                  name: "title",
+                  widget: "string",
+                  required: false,
```

Comment:
```suggestion
```

Title should required in settings unless we have a separate site information section with name (required) and description (optional) separate from SEO i.e. similar to Ghost 👇🏽 

![Screenshot from 2022-08-29 11-22-15](https://user-images.githubusercontent.com/1779590/187157544-89e8ba9e-db99-45b2-b1d4-c556b7618815.png)